frontend/src/components/Admin/AddUser/AddUser.tsx

```diff
@@ -149,6 +150,16 @@ export default function AddUser(): JSX.Element {
       return;
     }
 
+    // Check that no tpms are also advisors
+    if (member.role === 'tpm' && member.isAdvisor) {
+      Emitters.generalError.emit({
+        headerMsg: 'Invalid Role!',
+        contentMsg:
+          'TPM advisor is not a valid role. Please select a different role if this member is returning as an advisor.'
```

Comment:
nit: maybe add "Note that previous TPMs may become dev advisors."

Also, just sanity checking: business advisor, and PM advisor are both a thing?

Reply:
Ops Leads say so. I'm waiting to hear back from the other role leads about this too. We can probably double check during leads sync.

Reply:
Ok! Let's leave it then!